### ISSUE TITLE
SGE post_install script issue

### ISSUE THREAD

**Issue by dikim33** [oscar-cluster/oscar#215]
**Reported by bli on 25 Jul 2006 05:27 UTC**
Getting the following output with branch-5-0 r5215 with SUSE Linux 10.0 x86:

```
--> About to run /opt/oscar-5.0a1r5215M/packages/sge/scripts/post_install for sge
denied: can't delete master host "suse10-0-x86.ocg.org" from admin host list
"suse10-0-x86.ocg.org" does not exist in "hostlist" of "hostgroup"

root@suse10-0-x86.ocg.org modified "@allhosts" in host group list
Attribute name ("*") and/or value ("suse10-0-x86.ocg.org") not found
denied: execution host "suse10-0-x86.ocg.org" does not exist
```

**Comment by dikim33** [comment]
**Comment by bli on 29 Jul 2006 03:28 UTC**
The post_install script had a call to `hostname` which should be `hostname --long` on SUSE. Fix checked into trunk r5241.

This needs to be merged into branch-5-0 as well.

**Comment by dikim33** [comment]
**Comment by bli on 3 Aug 2006 01:42 UTC**
(In [5274]) This commit is a merge from trunk -> branch-5-0:
   svn merge -r5240:5241 https://svn.oscar.openclustergroup.org/svn/oscar/trunk .

This closes #216 for milestone OSCAR 5.0.

Original commit message:
Fix for #216.  To get the FQDN hostname, the command is `hostname --long` under SUSE.
